bpf/lib/nat.h

```diff
@@ -298,27 +298,43 @@ static __always_inline int snat_v4_track_local(struct __ctx_buff *ctx,
 	return 0;
 }
 
-static __always_inline int snat_v4_handle_mapping(struct __ctx_buff *ctx,
-						  struct ipv4_ct_tuple *tuple,
-						  struct ipv4_nat_entry **state,
-						  struct ipv4_nat_entry *tmp,
-						  enum nat_dir dir, __u32 off,
-						  const struct ipv4_nat_target *target)
+static __always_inline int snat_v4_nat_handle_mapping(struct __ctx_buff *ctx,
```

Comment:
Nit: It's a long line, I'd return after the return value.

```suggestion
static __always_inline int
snat_v4_nat_handle_mapping(struct __ctx_buff *ctx,
```